.all-contributorsrc

```diff
@@ -0,0 +1,40 @@
+{
+  "projectName": "fluffy-robot",
```

Comment:
this should be replaced with simulator i think :).

Reply:
Good catch!